doc/getting-started.rst

```diff
@@ -103,16 +113,16 @@ terminal with some embellishment.
    need to update the package index, you can do this by running ``cabal
    update``.
 
-In our ``myfirstapp.cabal`` file we'll update the ``build-depends`` attribute of
-the ``executable myfirstapp`` section to include ``haskell-say``:
+In our ``myapp.cabal`` file we'll update the ``build-depends`` attribute of
+the ``executable myapp`` section to include ``haskell-say``:
```

Comment:
```suggestion
the executable section to include ``haskell-say``:
```

I think this will be easier on the reader. It's clear what we mean here anyway I think.